.env

```diff
@@ -0,0 +1,53 @@
+# ---------------------------------
```

Comment:
Template configuration file.

All third party storage integrations can now be enabled/disabled by specifying/omitting their API keys.